java-checks/src/test/files/checks/spring/SpringEndpoints.java

```diff
@@ -0,0 +1,31 @@
+package test;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping(value = "/ex")
+public class SpringEndpoints {
```

Comment:
For sake of coverage shouldn't we have a method without an annotation that triggers the issue ?

Reply:
Did you mean "doesn't trigger the issue", or I don't understand?

Reply:
You're absolutely right, I forgot the negation.